docs/schema.md

```diff
@@ -654,6 +659,23 @@ HTTP Status Code: 200
     }
   }
 ```
+GET http://$GOHAN/[$namespace_prefix/]$prefix/$plural/$id&search_field=$id
```

Comment:
`/$id` is an example of GETting single resource, not multiple resources, so it's not a good place to explain substr match.